### PR TITLE
Updating to > 1.0 openai

### PR DESCRIPTION
**Pull request by danmcp**
Updating to a newer openai to not conflict with the cli

### CODE REVIEW

File: src/instructlab/eval/mmlu.py [hunk 1]
```diff
@@ -50,8 +50,7 @@ def run(self) -> tuple:
 
         individual_scores: dict = {}
         agg_score: float = 0.0
-        model_args = f"pretrained= {self.model_path}, dtype= {self.model_dtype}"
-
+        model_args = f"pretrained={self.model_path},dtype={self.model_dtype}"
```
Comment (danmcp) [inline src/instructlab/eval/mmlu.py:53]: @alimaredia @alinaryan FYI in case you hit this issue as well.

**Review by nathan-weinberg — APPROVED** [review]

File: requirements.txt [hunk 2]
```diff
@@ -2,7 +2,7 @@
 FastChat
 GitPython>=3.1.42,<4.0.0
 shortuuid
-openai<1.0.0
```
Comment (JamesKunstle) [inline requirements.txt:5]: are there any implications to this bump? or is it a clean reversioning?

**Review by JamesKunstle — APPROVED** [review]
everything changes here looks good if it works during manual testing. one question about versioning implications but not something that should block merging.

File: requirements.txt (hunk 2 above)
Comment (danmcp) [reply requirements.txt:5]: I might not be understanding the questions.  But there were several changes with the API that needed corresponding changes on our side that are in the commit.

**Review by alinaryan — APPROVED** [review]

**Pull request merged**